src/Hydra/Serializer/ErrorNormalizer.php

```diff
@@ -42,6 +43,10 @@ public function normalize($object, $format = null, array $context = [])
         $message = $object->getMessage();
         if ($this->debug) {
             $trace = $object->getTrace();
+        } else {
```

Comment:
If we're okay with modifying the status code in the `FlattenException`, like we currently do, then this works just fine.